src/model/rules.rs

```diff
@@ -0,0 +1,105 @@
+use super::grid::Grid;
+
+pub fn compute_new_state(grid: &Grid, x: usize, y: usize) -> bool {
```

Comment:
I think this function name is not very clear because `compute_new_state` seems to imply computing the state of the entire grid:

```suggestion
pub fn compute_cell_new_state(grid: &Grid, x: usize, y: usize) -> bool {
```

Reply:
I agree, I updated it.